ext/aws/module.lb

```diff
@@ -102,6 +102,9 @@ def build(env):
     env.copy("{}/portmacro.h".format(path), "freertos/inc/freertos/portmacro.h")
     # Copy the head_3.c file that just wraps the libc malloc
     env.copy("freertos/FreeRTOS/Source/portable/MemMang/heap_3.c", "freertos/heap_3.c")
+    # Copy the compiler support files
+    env.copy("freertos/FreeRTOS-Plus-TCP/portable/Compiler/GCC/pack_struct_start.h", "freertos_plus_tcp/include/pack_struct_start.h")
+    env.copy("freertos/FreeRTOS-Plus-TCP/portable/Compiler/GCC/pack_struct_end.h", "freertos_plus_tcp/include/pack_struct_end.h")
```

Comment:
This has to go into the `:freertos:tcp` submodule above.
You can add the pack files to the ignore list:
```py
        env.copy("freertos/FreeRTOS-Plus-TCP", "freertos_plus_tcp",
            ignore=env.ignore_files("BufferAllocation_1.c"))
```

Reply:
Thanks, also resolved the duplicates